src/System.Security.Cryptography.X509Certificates/src/project.json

```diff
@@ -5,6 +5,7 @@
     "System.Diagnostics.Debug": "4.0.10",
     "System.Globalization.Calendars": "4.0.0",
     "System.IO.FileSystem": "4.0.0",
+    "System.IO.FileSystem.Watcher": "4.0.0-beta-*",
```

Comment:
System.IO.FileSystem.Watcher doesn't have a stable version published, so the -beta-* is required.